php/commands/site.php

```diff
@@ -635,35 +635,25 @@ public function clone_env($args, $assoc_args) {
 
      if ($db) {
        print "Cloning database ... ";
-       $this->cloneObject( $to_env, $from_env, $site_id, 'database');
+       $workflow = $site->workflows->create("clone_database", array(
+         'environment' => $to_env,
+         'params' => array('from_environment' => $from_env)
+       ));
+       $workflow->wait();
      }
 
      if ($files) {
       print "Cloning files ... ";
-      $this->cloneObject( $to_env, $from_env, $site_id, 'files');
+      $workflow = $site->workflows->create("clone_files", array(
+        'environment' => $to_env,
+        'params' => array('from_environment' => $from_env)
+      ));
+      $workflow->wait();
```

Comment:
I think that would add confusion to what is happening. I'd rather keep them somewhat concrete. I think workflows are abstract enough.